ketcher-autotests/tests/Macromolecule-editor/Rectangle-Selection-Tool/rectangle-selection-tool.spec.ts

```diff
@@ -69,4 +70,52 @@ test.describe('Rectangle Selection Tool', () => {
       path: 'tests/Macromolecule-editor/screenshots/rectangle-selection-tool3.png',
     });
   });
+
+  test.skip('Move monomer bonded with another monomers', async ({ page }) => {
+    /* 
+    Test case: #2367 - move items on the canvas
+    Description: check ability to move items on the canvas
+    */
+
+    await turnOnMacromoleculesEditor(page);
+
+    // Choose peptide
+    await page.getByText('Tza').click();
+
+    // Create 4 peptides on canvas
+    await page.mouse.click(300, 300);
+    await page.mouse.click(400, 400);
+    await page.mouse.click(500, 500);
+    await page.mouse.click(600, 600);
+
+    // Get 4 peptides locators
+    const peptides = await page.getByText('Tza').locator('..');
+    const peptide1 = peptides.nth(0);
+    const peptide2 = peptides.nth(1);
+    const peptide3 = peptides.nth(2);
+    const peptide4 = peptides.nth(3);
+
+    // Select bond tool
+    await selectSingleBondTool(page);
+
+    // Create bonds between peptides
+    await bondTwoMonomers(page, peptide1, peptide2);
+    await bondTwoMonomers(page, peptide3, peptide2);
+    await bondTwoMonomers(page, peptide3, peptide4);
+
+    await page.screenshot({
```

Comment:
We have function takeEditorScreenshot(page) which takes screenshots of result structure, name it according to test function name and place into folder - you may reuse it here and on 117 line

Reply:
I decided to use this function here because I saw `screenshots` folder and I wanted to put it the same place. 
I replaced the way to do screenshots. Now I use 'takeEditorScreenshot'